tests/test_data_iterator.py

```diff
@@ -0,0 +1,90 @@
+from typing import List
+
+import pandas as pd
+import pandas.testing as pd_test
+import pytest
+
+from packages.python.accern_data import create_data_client
+from packages.python.accern_data.util import EXAMPLE_URL
+
+
+@pytest.mark.parametrize(
+    "sheet_mode, chunk_size",
+    [
+        ("df", 1),
+        ("df", 5),
+        ("df", 1000),
+        ("csv", 1),
```

Comment:
you mean csv and json?
no need to test both df *and* csv
you can switch it up (for example "csv" if chunk_size % 2 == 0 else "df" idk)
also, can you do one test with default chunk_size